mcweb/backend/search/views.py

```diff
@@ -472,8 +472,8 @@ def providers(request):
 
 def add_ratios(words_data):
     for word in words_data:
-        if "ratio" not in word:
-            word["ratio"] = word['count'] / 1000
+        if "term_ratio" not in word:
+            word["term_ratio"] = word['count'] / 1000
```

Comment:
What happens if the wayback provider returns data?  A quick and *DIRTY* fix would be to make the Wayback words method throw an exception